src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -1271,6 +1271,20 @@ public static void DCJS_SerializationEvents()
         Assert.True(output.OnDeserializedMethodInvoked, "output.OnDeserializedMethodInvoked is false");
     }
 
+    [Fact]
+    public static void DCJS_DeserializeEmptyString()
+    {
+        var exception = Assert.Throws<XmlException>(() =>
+        {
+            var serializer = new DataContractSerializer(typeof(object));
```

Comment:
Yes, this is a typo. After fixing this I've realized this issue is only for DCS. DCJS throws SerializationException which aligns with the behavior on Desktop.